### ISSUE TITLE
Tiny YOLO does not detect objects from most of training images

### ISSUE THREAD

**Issue by YanzeeNongsiej** [YanzeeNongsiej/iainong#1]
I have been training tiny yolo on a dataset of 99 images and only one class. The loss and average loss oscillates between 0.1- 0.5 . It did not go below that for 2 days now.I'm currently at iteration 15700ish . I tried decreasing the learning rate from 1e-5 to 1e-6 but nothing worked. This is the second time i trained on this dataset. Idk what the problem is. my dataset probably?
![Untitled](https://user-images.githubusercontent.com/18144318/68408994-5d53d800-01ac-11ea-9a93-bdbdeb2300d6.png)
